Chapters/ThreePillarsOfSpec/ThreePillarsOfSpec.md

```diff
@@ -32,37 +29,40 @@ This is exactly what we do hereafter.
 First, we create a new presenter class.
 
 ```
-SpPresenter << #SpMethodLister
-    slots: { #aClass . #list};
-    package: 'Spe2Book'
+SpPresenter << #MethodLister
+    slots: { #sourceClass . #list};
```

Comment:
`aClass` is a strange name for an instance variable. It sounds more like a method argument. `sourceClass` is a better name for an instance variable. Moreover, it expresses intent better.